### DISCUSSION TITLE
Welcome to DSS Discussions!

### DISCUSSION THREAD

**Discussion by perdrix52** [deepskystacker/DSS#131] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas

**Comment by williamli9300** [comment]
It would be nice to be able to reposition the file list tab, e.g. to the right of the image preview (example at the end), or undock it or the image preview window (i.e. view it in a separate always-on-top window). I work with a widescreen monitor, so placing the file list on the right would allow me to view the image with the full height of my screen (that way i can compare subs without having to re-zoom each time) but still be able to see the file list. On non-widescreen monitors, having an image list tab that you can undock would allow you to be able to see the image in full-size/with your full screen real estate without having to re-zoom each time. Another cool idea would be to be able to maintain zoom while viewing different pictures (e.g. workflow: clicking one sub to preview, zooming in, then clicking another sub, then switching back and forth to be able to visually inspect stars etc. at a magnification). Some other related ideas are a magnification menu (100%, 200%, etc. like the one found in Adobe Acrobat) and the ability to zoom or set preview to fit with a shortcut key (+, -, 0 etc) for ease of preview.
```
----------------------------------------------------------------
|  menu  |                       |       |      |       |      |
|  reg.  |          image        | file  |filt. |  type | fwhm |
| stack  |                       |       |      |       |      |
|   etc  |                       |       |      |       |  etc |
----------------------------------------------------------------
```

**Comment by perdrix52** [comment]
Thank you for the suggestion

**Reply by perdrix52** [reply]
Here's a screen shot from the current development code which I think you will like:

![image](https://user-images.githubusercontent.com/20211762/196000520-4a6c4b49-9c8a-4676-b34c-817306fd369d.png)
